scenarios/evaluate-safety/README.md

```diff
@@ -0,0 +1,29 @@
+---
+page_type: sample
+languages:
+- language1
+- language2
+products:
+- ai-services
+- azure-openai
+description: Evaluate Protected Material and Indirect Attack Jailbreak
+---
+
+## Evaluate Protected Material and Indirect Attack Jailbreak
+
+### Overview
+
+This notebook walks through how to generate a simulated conversation targeting a deployed AzureOpenAI model and then evaluate that conversation dataset for Protected Material and Indirect Attack Jailbreak vulnerability. It also references the prompt filtering capabilities of Azure AI to help identify and mitigate these vulnerabilities in your AI system.
```

Comment:
Rewrite: "This notebook walks through how to generate a simulated conversation targeting a deployed AzureOpenAI model and then evaluate that conversation test dataset for Protected Material and Indirect Attack Jailbreak (also know as XPIA or cross-domain prompt injected attack) vulnerability. It also references Azure AI Content Safety service's prompt filtering capabilities to help identify and mitigate these vulnerabilities in your AI system."